PendingReleaseNotes

```diff
@@ -1,5 +1,8 @@
 >=18.0.0
 
+* The snap-schedule mgr module now retains one less snapshot than the number
```

Comment:
Normally, new additions are done at the section end and include a prefix that its related to cephfs.

Reply:
updated PendingReleaseNotes appropriately